aten/src/ATen/native/Distributions.cpp

```diff
@@ -600,4 +576,72 @@ Tensor multinomial(
   return result;
 }
 
-}} // namespace at::native
+} // namespace at::native
+
+namespace meta {
+
+// ==================================================== Normal ========================================================
+
+TORCH_META_FUNC2(normal, float_Tensor) (
+  double mean,
+  const Tensor& std,
+  c10::optional<Generator> gen
+) {
+  set_output(std.sizes(), std.options());
+}
+
+TORCH_META_FUNC2(normal, Tensor_float) (
+  const Tensor& mean,
+  double std,
+  c10::optional<Generator> gen
+) {
+  set_output(mean.sizes(), mean.options());
+}
+
+TORCH_META_FUNC2(normal, Tensor_Tensor) (
+  Tensor const& mean,
+  Tensor const& std,
+  c10::optional<Generator> gen
+) {
+  auto shape = at::infer_size(mean.sizes(), std.sizes());
```

Comment:
Just so we don't forget: probably we want to do something like what [`resize_output_for_normal`](https://github.com/pytorch/pytorch/blob/bac7d8c197ce6a2c9e4d561c6598507ffb08564f/aten/src/ATen/native/DistributionTemplates.h#L167) does here, inside META. Since we still have the same problem as the dtype checks, we should probably wait for Brian.